tests/modules/controlfreec/freec/test.yml

```diff
@@ -20,4 +20,26 @@
     - path: output/controlfreec/test2.mpileup.gz_sample.cpn
       md5sum: c80dad58a77b1d7ba6d273999f4b4b4b
     - path: output/controlfreec/versions.yml
-      md5sum: 3ab250a2ab3be22628124c7c65324651
+
+- name: controlfreec test_controlfreec_freec
+  command: nextflow run tests/modules/controlfreec/freec -entry test_controlfreec_freec_single -c tests/config/nextflow.config -stub-run
+  tags:
+    - controlfreec
+    - controlfreec/freec
+  files:
+    - path: output/controlfreec/config.txt
+    - path: output/controlfreec/test.mpileup.gz_control.cpn
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
I think we don't need md5sum for empty files

Reply:
ok